docs/mergeop/1_mergeop.md

```diff
@@ -0,0 +1,255 @@
+# **MergeOp**
+
+## **Requirements**
+
```

Comment:
It'd be great if we have a description about the high-level use-cases/benefits by MergeOp. (e.g. which Dockerfile instructions are expected to be optimized by this?)

Reply:
Good call, I had only discussed it with people familiar with the use cases before this and forgot to include them here :-)

Added a Use Cases section.